client/store/order.js

```diff
@@ -0,0 +1,108 @@
+import axios from 'axios'
+
+/**
+ * ACTION TYPES
+ **/
+const GET_ONE_ORDER = 'GET_ONE_ORDER'
+const LOADING = 'LOADING'
+const DELETE_LINE_ITEM = 'DELETE_LINE_ITEM'
+const CHECK_OUT = 'CHECK_OUT'
+/**
+ * INITIAL STATE
+ **/
+const initialState = {
+  defaultOrder: {},
+  defaultLoading: true,
+  order: {},
+}
+
+/**
+ * ACTION CREATORS
+ **/
+const getOneOrder = order => ({
+  type: GET_ONE_ORDER,
+  order
+})
+
+const loading = status => ({
+  type: LOADING,
+  loading: status
+})
+
+const deleteLineItem = (order) => ({
+  type: DELETE_LINE_ITEM,
+  order
+})
+
+const checkout = (order) => ({
+  type: CHECK_OUT,
+  order
+})
+
+/**
+ * THUNK CREATORS
+ **/
+// All this does is get an order by orderId from the backend.
+const orderGetter = (orderId) =>
+  {
+    return axios.get(`/api/orders/${orderId}`)
+      .then(order => order.data)
```

Comment:
Should we be dispatching an event to the store

Reply:
This is just a helper function so I can keep my code more DRY!